src/main/java/fizzbuzz/FizzBuzz.java

```diff
@@ -0,0 +1,12 @@
+package fizzbuzz;
+
+public class FizzBuzz {
+    public static void main(String[] args) {
+        for (int i = 1; i <= 100; i++) {
```

Comment:
to powinno być zrobione w taki sposób, że gdybym np. chciał zakres od 300 - 1000 to nie musisałbym pisać tego kodu od nowa, tylko musiałbym wywołać metode z innymi parametrami. podobnie z warunkami